components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -567,8 +568,10 @@ - (void)movePlaceholderToUp:(BOOL)isToUp {
 
 - (void)updatePlaceholderAnimationConstraints:(BOOL)isToUp {
   if (isToUp) {
-    CGPoint destination = [self floatingPlaceholderDestination];
+    UIOffset destination = [self floatingPlaceholderOffset];
+    UIEdgeInsets insets = self.textInput.textInsets;
 
+    CGFloat horizontalLeading = insets.left - destination.horizontal;
```

Comment:
Do you need to use `insets.right` for RTL layouts or is it already flipped?

Reply:
Already flipped. I've updated the documentation to explain that.